docs/admin/architectures/index.md

```diff
@@ -171,3 +171,167 @@ Database:
+recommend referencing public cloud providers such as AWS, GCP, and Azure for
+guidance on optimal configurations. A reasonable approach involves using scaling
+formulas based on factors like CPU, memory, and the number of users.
+
+While the minimum requirements specify 1 CPU core and 2 GB of memory per
+`coderd` replica, it is recommended to allocate additional resources to ensure
+deployment stability.
+
+#### CPU and memory usage
+
+The memory consumption may increase with enabled agent stats collection by the
+Prometheus metrics aggregator (optional).
+
+Enabling direct connections between users and workspace agents (apps or SSH
+traffic) can help prevent an increase in CPU usage. It is recommended to keep
+this option enabled unless there are compelling reasons to disable it.
+
+Inactive users do not consume Coder resources.
+
+#### Scaling formula
+
+When determining scaling requirements, consider the following factors:
+
+- `1 vCPU x 2 GB memory x 250 users`: A reasonable formula to determine resource
+  allocation based on the number of users and their expected usage patterns.
+- API latency/response time: Monitor API latency and response times to ensure
+  optimal performance under varying loads.
+- Average number of HTTP requests: Track the average number of HTTP requests to
+  gauge system usage and identify potential bottlenecks.
+
+**HTTP API latency**
+
+For a reliable Coder deployment dealing with medium to high loads, it's
+important that API calls for workspace/template queries and workspace build
+operations respond within 300 ms. However, API template insights calls, which
+involve browsing workspace agent stats and user activity data, may require more
+time.
+
+Also, if the Coder deployment expects traffic from developers spread across the
+globe, keep in mind that customer-facing latency might be higher because of the
+distance between users and the load balancer.
+
+**Node Autoscaling**
+
+We recommend disabling the autoscaling for `coderd` nodes. Autoscaling can cause
+interruptions for user connections, see [Autoscaling](../scale.md#autoscaling)
+for more details.
+
+### Control plane: provisionerd
+
+Each provisioner can run a single concurrent workspace build. For example,
+running 10 provisioner containers will allow 10 users to start workspaces at the
+same time.
+
+By default, the Coder server runs built-in provisioner daemons, but the
+_Enterprise_ Coder release allows for running external provisioners to separate
+the load caused by workspace provisioning on the `coderd` nodes.
+
+#### Scaling formula
+
+When determining scaling requirements, consider the following factors:
+
+- `1 vCPU x 1 GB memory x 2 concurrent workspace build`: A formula to determine
+  resource allocation based on the number of concurrent workspace builds, and
+  standard complexity of a Terraform template. _Rule of thumb_: the more
+  provisioners are free/available, the more concurrent workspace builds can be
+  performed.
```

Comment:
This aligns with my experience; a single provisionerd tends to not go much above 0.1C / 512MB.

Reply:
Should I decrease these numbers or leave as is?

Reply:
I think you can leave as-is.